app/generator/publish.js

```diff
@@ -0,0 +1,25 @@
+const { getOutboundBlobClient } = require('../storage')
```

Comment:
missing tests

Reply:
Similar to other comment, all behaviour is covered in higher level tests.   It's also pretty difficult to setup a test at this level as the event emitters are awkward.  I felt it was a lot of effort to cover nothing new.

What do you think?